CHANGELOG.md

```diff
@@ -1,5 +1,70 @@
 ### [Versions](https://material-ui.com/versions/)
 
+## 5.0.0-beta.3
+
+<!-- generated comparing v5.0.0-beta.2..next -->
+
+_Aug 6, 2021_
+
+Big thanks to the 14 contributors who made this release possible. Here are some highlights ✨:
+
+- <!-- 28 --> ✨ Improve `jss-to-styled` codemod to support `createStyles` usage (#27578) @mnajdova
+- <!-- 23 --> 📦 We make a good progress on the Unstyled package with `FormControl` and `useAutocomplete` (#27485) @michaldudak
```

Comment:
```suggestion
- <!-- 23 --> 📦 Introduced new `FormControlUnstyled` component (#27485) @michaldudak
```

Reply:
I don't think that moving the hook to the unstyled package is a highlight, but I may be wrong :)

Reply:
I agree :)